src/EVM/Effects.hs

```diff
@@ -46,6 +46,7 @@ data Config = Config
   , maxBranch     :: Int
   , promiseNoReent   :: Bool
   , maxBufSize       :: Int
+  , maxExplore     :: Maybe Int
```

Comment:
Can you explain what is the difference between `maxBranch` and `maxExplore`?

Reply:
Good point actually! Thanks, you are really good at this. This needs clarification, and likely some rewrite, you are right. It's a mish-mash. Let me think about it a bit!

Reply:
OK, I now explained it better in `--help`  and I wrote a description in the `common-options.md` as well. Hopefully it's clearer now!